README.md

```
@@ -39,12 +39,13 @@ Mobile-use is a powerful, open-source AI agent that controls your Android or IOS
 <p align="center">
-  <img src="./doc/benchmark.jpg" alt="Project banner" />
+  <a href="https://minitap.ai/benchmark">
+    <img src="https://files.peachworlds.com/website/753680a0-c383-451e-9800-703d04489ea9/comparison.png" alt="Project banner" />
+  </a>
 </p>
 
-We are global number 1 Opensource pass@1 on the AndroidWorld benchmark.
-
-More info here: https://minitap.ai/research/mobile-ai-agents-benchmark
+We stand among the top performers in the AndroidWorld benchmark.
+More info here: https://minitap.ai/benchmark
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)